docs/platform/linode-gpu/getting-started-with-gpu/index.md

```diff
@@ -113,26 +143,46 @@ Installation on CentOS and RHEL operating systems will fail following this as th
 
 ## Manual Install
 
-This section will walk you through the process of downloading and installing the latest NVIDIA driver on Debian 9. This process can be completed similarly on a Distro of your choice if needed:
+This section will walk you through the process of downloading and installing the latest NVIDIA driver on Debian 9. This process can also be completed similarly on another distro of your choice if needed:
+
+1. Visit NVIDIA's [Driver Downloads Page](https://www.nvidia.com/Download/index.aspx?lang=en-us).
+
+1. Make sure that the options from the drop-down menus reflect the following values:
 
-1. Find the latest Driver from NVIDIA's [Driver Downloads Page](https://www.nvidia.com/Download/index.aspx?lang=en-us).
+    | Prompt | Selection |
+    |--------|-----------|
+    | Product Type | Quadro |
+    | Product Series | Quadro RTX Series |
+    | Product | Quadro RTX 8000 |
+    | Operating System | Linux 64-bit |
+    | Download Type | Optimal Driver for Enterprise (ODE) |
+    | Download Type | Linux Long Lived Driver |
+    | Language | English (US) |
 
-1. Make sure that the options from the drop-down menus reflect the following:
+    The form will look as follows when completed:
 
     ![Driver Download](driver.png "Driver Download")
 
-1. Select "Search", "Download", and download the Linux driver that appears on the following screen to your Linode.
+1. Click the **Search** button, and a page will appear that shows information about the driver. Click the green **Download** button on this page. The file will not download to your computer; instead, you will be taken to another download confirmation page.
+
+1. Copy the link for the driver installer script from the green **Download** button on this page:
+
+    ![Copy Download Link](copy-driver-download-link.png "Right click to copy the download link for the installer")
+
+1.  On your Linode, enter the `wget` command and paste in the download link you copied. This example shows the syntax for the command, but you should make sure to use the download link you copied from NVIDIA's site:
+
+        wget http://us.download.nvidia.com/XFree86/Linux-x86_64/430.14/NVIDIA-Linux-x86_64-430.14.run
 
-1. Run the script to install the driver. Follow the prompts as necessary:
+1. After wget completes, run your version of the installer script on your Linode. Follow the prompts as necessary:
 
-        sudo bash NVIDIA-Linux-x86_64-430.14.run
+        sudo bash NVIDIA-Linux-x86_64-*.run
 
 1. Select `OK` and `Yes` for all prompts as they appear.
 
-1. Once the installer has completed, use `nvidia-smi` to make sure that you're currently using your NVIDIA GPU with it's associated driver:
+1. Once the installer has completed, use `nvidia-smi` to make sure that you're currently using your NVIDIA GPU with its associated driver:
 
         nvidia-smi
```

Comment:
I think we should have a Next Steps section that follows the installation instructions. Maybe this could link to the other GPU guide we're writing when it's finished